analysis/sextractor_analysis.py

```diff
@@ -3,9 +3,9 @@
 import pylab as plt
 import pandas as pd
 
-DATADIR = "/Users/alexandre/work/lsst/deblending/coin/candels-blender/output-s_666-n_30000"
-CATDIR = "/Users/alexandre/work/lsst/deblending/coin/candels-blender/output-s_666-n_30000"
-FINALCATDIR = "/Users/alexandre/work/lsst/deblending/coin/catalog_results"
+DATADIR = "<path-to-data-dir>"
+CATDIR = "<path-to-data-dir>"
+FINALCATDIR = "<path-to-catalog-results>"
```

Comment:
thanks ! 😄

Reply:
What do you think about using environment variables for this ?

```python
import os 
os.getenv("COINBLEND_DATADIR")
```

Reply:
very welcome, thanks to you! 

> What do you think about using environment variables for this ?
> 
> ```python
> import os 
> os.getenv("COINBLEND_DATADIR")
> ```

Right, environment variables would be much more handy, I totally agree!

Reply:
Would you like to do these changes yourself ?

Reply:
I can probably do it later today, yes :)

Reply:
Hi, just send another request - the datadir is now set via datadir=os.getenv("COINBLEND_DATADIR") as proposed, and I added a note in the README to export COINBLEND_DATADIR=<path-to-datadir>. There are two issues left in sextractor_analysis.py, the CATDIR in l.7 and l.147 that points to a data file (I'm afraid to touch/delete), and the if{} in l.125-130 that should be obsolete (together with the mix variable).

Reply:
Indeed I saw the obsolete `if` statement. You can remove it as well.